docs/zh_cn/overview.md

```diff
@@ -1,3 +1,50 @@
-# Overview
+# 总体介绍
 
-- [Overview](#overview)
+- [总体介绍](#总体介绍)
+  - [自监督学习](#自监督学习)
+  - [MMSelfSup的架构设计](#mmselfsup的架构设计)
+  - [MMSelfSup的上手路线图](#mmselfsup的上手路线图)
+    - [使用MMSelfSup玩转自监督实验](#使用mmselfsup玩转自监督实验)
+    - [基于MMSelfSup学习自监督算法](#基于mmselfsup学习自监督算法)
+
+在本文档中，我们将对[MMSelfSup](https://github.com/open-mmlab/mmselfsup)进行整体介绍。
+我们首先会对自监督学习(Self-supervised Learning)的基本概念进行回顾，然后简单介绍整个MMSelfSup的基本架构。最后，我们将给出MMSelfSup的上手路线图，帮助大家更快的使用MMSelfSup助力自己的科学研究和项目实践。
+
+## 自监督学习
+
+自监督学习(Self-supervised learning, SSL)是一种极具潜力的学习范式，它旨在使用海量的无标注数据来进行表征学习。在SSL中，我们通过构造合理的预训练任务（可自动生成标注，即自监督）来进行模型的训练，学习到一个具有强大建模能力的预训练模型。基于自监督学习获得的训练模型，我们可以提升各类下游视觉任务(图像分类，物体检测，语义分割等)的性能。
+
+过去几年里，自监督学习的研究获得了快速发展，整个学术社区涌现了一大批优秀的自监督学习算法。我们旨在将MMSelfSup打造成为一个功能强大，方便易用的开源算法库，助力学术研究和工程实践。接下来，我们将介绍MMSelfSup的架构设计。
+
+## MMSelfSup的架构设计
+
+与其他OpenMMLab的项目类似，MMSelfSup基于模块化设计的准则，整体的架构设计如下图所示：
+
+<div align="center">
+  <img src="https://user-images.githubusercontent.com/36138628/187197848-e518052f-b75e-4981-bed7-d71dc1a5a0fb.jpg" width="500"/>
```

Comment:
Update with this pic? also English version
![img_v2_bfd6a517-1616-43ce-a40a-aab77615d25g](https://user-images.githubusercontent.com/36138628/199443908-e7fd3670-108b-46eb-b200-d76f25e5621b.jpg)